code/src/Music.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import data from './data.json'
+
+
+const oldList = data.albums.items 
+
+export const albumNew = []
+
+const ArtistsName = (oldList) => {
+
+    const newList  = []
+
+    oldList.forEach(item => {
+        newList.push({name: item.name, link: item.external_urls.spotify})
+    })
+
+    //let artistWord = newList.toString()
+    //artistWord.replace(",", ", ")
+    return  newList 
+
+}
+
+oldList.forEach((item => {
+    const num = item.artists.length
```

Comment:
What does `num` stands for? Maybe another, more describing word would suit good here?